components/SectionHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from '../styles/components/SectionHeader.module.scss';
+
+type Location = 'justify-start' | 'justify-end';
+
+interface Props {
+  title: string;
+  backgroundColor?: string;
+  foregroundColor?: string;
+  textColor?: string;
+  location?: Location;
+}
+
+export default function SectionHeader({
+  title,
+  backgroundColor = 'bg-color-1',
+  foregroundColor = 'bg-dark-gray',
+  textColor = 'text-color-2',
+  location = 'justify-start',
```

Comment:
I think location would still be seperate from the styles but can we change this to an enum of like `rightAligned | LeftAligned` its far more informative imo.

Reply:
Fixed